microsoft-edge/webview2/how-to/remote-debugging.md

```diff
@@ -97,6 +97,84 @@ Attach Microsoft Edge DevTools remotely to a WebView2 WinUI 2 (UWP) app as follo
 Now you can use Microsoft Edge DevTools to inspect and debug the WebView2 control.
 
 
+<!-- ====================================================================== -->
+## Debugging Hololens 2 WebView2 WinUI 2 (UWP) apps
+
+Attach Microsoft Edge DevTools remotely to a Hololens 2 WebView2 WinUI 2 (UWP) app as follows:
+<!-- steps 1 & 2: -->

```

Comment:
Global: If the correct steps got copied from main section to here, probably now remove hidden comments like:
```
<!-- steps 1 & 2: -->
<!-- / steps 1 & 2 -->
<!-- steps 8-12: -->
```
which are probably 1 too high now anyway